<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sudoku-Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GameArea" name="gameArea" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>860</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>640</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_New_Game">
    <property name="title">
     <string>&amp;New Game</string>
    </property>
    <addaction name="actionLv_0"/>
    <addaction name="actionLv_1"/>
    <addaction name="actionLv_2"/>
    <addaction name="actionLv_3"/>
    <addaction name="actionLv_4"/>
    <addaction name="actionLv_5"/>
    <addaction name="actionLv_6"/>
    <addaction name="actionLv_7"/>
    <addaction name="actionLv_8"/>
    <addaction name="actionLv_9"/>
    <addaction name="actionLv_10"/>
    <addaction name="separator"/>
    <addaction name="actionRandom"/>
   </widget>
   <addaction name="menu_New_Game"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLv_1">
   <property name="text">
    <string>Lv.&amp;1</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.1</string>
   </property>
  </action>
  <action name="actionLv_2">
   <property name="text">
    <string>Lv.&amp;2</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.2</string>
   </property>
  </action>
  <action name="actionLv_3">
   <property name="text">
    <string>Lv.&amp;3</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.3</string>
   </property>
  </action>
  <action name="actionLv_4">
   <property name="text">
    <string>Lv.&amp;4</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.4</string>
   </property>
  </action>
  <action name="actionLv_5">
   <property name="text">
    <string>Lv.&amp;5</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.5</string>
   </property>
  </action>
  <action name="actionLv_6">
   <property name="text">
    <string>Lv.&amp;6</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.6</string>
   </property>
  </action>
  <action name="actionLv_7">
   <property name="text">
    <string>Lv.&amp;7</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.7</string>
   </property>
  </action>
  <action name="actionLv_8">
   <property name="text">
    <string>Lv.&amp;8</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.8</string>
   </property>
  </action>
  <action name="actionLv_9">
   <property name="text">
    <string>Lv.&amp;9</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.9</string>
   </property>
  </action>
  <action name="actionLv_10">
   <property name="text">
    <string>Lv.&amp;10</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.10</string>
   </property>
  </action>
  <action name="actionLv_0">
   <property name="text">
    <string>Lv.&amp;0</string>
   </property>
   <property name="toolTip">
    <string>Load Lv.0</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="text">
    <string>Random</string>
   </property>
   <property name="toolTip">
    <string>Load a new random game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GameArea</class>
   <extends>QWidget</extends>
   <header>gamearea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
